opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+from enum import Enum
+
+from .. import Span, SpanProcessor
+
+
+class ExportResult(Enum):
+    SUCCESS = 0
+    FAILED_RETRYABLE = 1
+    FAILED_NONE_RETRYABLE = 2
+
+
+class Exporter:
+    """Interface that allows different services to export recorded spans in
+    its own format.
+
+    To export data this MUST be registered to the :class`..Tracer` using a
+    `SimpleSpanProcessor` or a `BatchSpanProcessor`.
+    """
+
+    def export(self, spans: typing.Sequence[Span]) -> "ExportResult":
+        """Exports a batch of telemetry data.
+
+        Args:
+            spans: The list of `Span`s to be exported
+
+        Returns:
+            The result of the export
+        """
+
+    def shutdown(self) -> None:
+        """Shuts down the exporter.
+
+        Called when the SDK is shut down.
+        """
+
+
+class SimpleSpanProcessor(SpanProcessor):
```

Comment:
What is the intented purpose of this class? Testing? Demo? Prototype until we have a more elaborate transport? Default implementation unless an user configures a more elaborate transport?

Reply:
I think this class is more for testing and demo purposes, the [BatchProcessor](https://github.com/open-telemetry/opentelemetry-specification/blob/9a0d656a817da2a2d92dabed732caaadfb470f3c/specification/sdk-span-processors.md#batching-processor) should be the choice when doing real stuff. 

According to my understanding these two classes are like glue code between span processors and exporters.

Reply:
That's correct, `SimpleExportSpanProcessor` is offered as a very simple option for testing and similar scenarios.